scripts/commands/checkExternalLinks.ts

```diff
@@ -0,0 +1,78 @@
+// This code is a Qiskit project.
+//
+// (C) Copyright IBM 2024.
+//
+// This code is licensed under the Apache License, Version 2.0. You may
+// obtain a copy of this license in the LICENSE file in the root directory
+// of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+//
+// Any modifications or derivative works of this code must retain this
+// copyright notice, and modified files need to carry a notice indicating
+// that they have been altered from the originals.
+
+import { globby } from "globby";
+import yargs from "yargs/yargs";
+import { hideBin } from "yargs/helpers";
+
+import { ExternalLink } from "../lib/links/ExternalLink";
+import { parseFile } from "../lib/links/extractLinks";
+import { addLinksToMap } from "../lib/links/FileBatch";
+
+interface Arguments {
+  [x: string]: unknown;
+  globs?: string[];
+}
+
+const readArgs = (): Arguments => {
+  return yargs(hideBin(process.argv))
+    .version(false)
+    .command("$0 [globs..]", "")
```

Comment:
Nice command! I didn't know this was possible, TIL

Reply:
Same. The search phrase was "yargs variadic positional arguments"